File: src/main/java/co/com/fsighttech/imp/issuing/bo/configuration/ProductVariableData.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo.configuration;

import java.util.List;

import co.com.fsighttech.imp.issuing.bo.core.Product;
import lombok.Getter;
import lombok.Setter;

/**
 * this class represents the variables data for the product 
 * 
 * @author jtorres
 *
 */
@Getter
@Setter
public class ProductVariableData {
	
	private Long id;
	private Product product;
	private List<MasterVariableData> masterVariableDataList;
	

}

File: bin/static/src/app/issuing-module/shared/model/configuration/product-variable-data.ts
import { JsonMember, JsonObject } from 'typedjson-npm/src/typed-json';

import { MasterVariableData } from '../configuration/master-variable-data';
import { InputProperties } from '../configuration/input-properties';
import { Product } from '../core/product';

/**
* this object represents a product variable data
*
* @author ccasallas
*/
@JsonObject
export class ProductVariableData {
    @JsonMember
    id: number;
    @JsonMember
    product: Product;
    @JsonMember({elements: MasterVariableData})
    masterVariableDataList: MasterVariableData[];
}